ReadMe.md

```diff
@@ -71,26 +84,34 @@ All test classes inherit from 'GenericTest' which implements some basic schema c
 Each manually defined test is expected to be defined as a method starting with 'test_'. This will allow it to be automatically discovered and run by the test suite. The return type for each test must be the result of calling one of the following methods on an object of class Test. An example is included below:
 
 ```python
-from TestHelper import Test
+from TestResult import Test
 
 def test_my_stuff(self):
     test = Test("My test description")
-
+    # Test code
     if test_passed:
-        # Pass the test
-        return test.PASS()
+        return test.PASS("Successful test case.")
     elif test_failed:
-        # Fail the test
-        return test.FAIL("Reason for failure")
+        return test.FAIL("Required feature of the specification has been found to be "

```

Comment:
Looks good. My only query is around these examples as they seem to repeat the text from above. Given that the string here is intended to provide additional data to assist with debugging errors or explaining the failure it might be better to provide examples of that sort of thing?

Reply:
I've had a go at this. Would you mind having another look? Thanks!